app/services/idv/session.rb

```diff
@@ -94,6 +94,13 @@ class Session
       welcome_visited
     ].freeze
 
+    SUPPORTED_ID_DOC_TYPES = [
+      'passport',
+      'passport_card',
```

Comment:
Based on https://github.com/18F/identity-idp/pull/12310, I thought we didn't support `passport_card`?